SYCL/BFloat16/bfloat16_type.cpp

```diff
@@ -1,10 +1,8 @@
-// UNSUPPORTED: cuda || hip
+// UNSUPPORTED: hip
 // RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Currently this will only check the software emulation cases are compiled and run correctly for the CUDA backend case. You can at least check the native versions are also compiling correctly if you add:
```suggestion
// RUN: %if cuda %{%clangxx -fsycl -fsycl-targets=%sycl_triple -Xsycl-target-backend --cuda-gpu-arch=sm_80 %s -o %t.out %}
// TODO enable the below when CI supports >=sm_80
// RUNx: %if cuda %{%GPU_RUN_PLACEHOLDER %t.out %}
// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Also I think you could now delete the bfloat16_type_cuda.cpp test since it is really just duplicating this test.

Reply:
Does the test pool contain any systems with CUDA devices that are >= sm_80?  If so, can we write a separate test that uses REQUIRE to enable a test only on that system?  This would allow us to compile and run the test with native support.

Reply:
Last I heard there was only sm75. Is there an advantage to having a separate test with REQUIRE for such as case? As opposed to just adding %if cases to an existing file? I'm just trying to understand the CI better.

Reply:
I've reenabled the CUDA execution option. I don't know if REQUIRE causes the test to be run on a pre-selected machine. Will find out.